core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerDAO.scala

```diff
@@ -104,23 +105,51 @@ class BillingProfileManagerDAOImpl(samDAO: SamDAO,
       userInfo
     ).flatMap {
       case true =>
-        Future.successful(
-          Seq(
+        val profileApi = apiClientProvider.getProfileApi(userInfo.accessToken.token)
+        val fullBillingProjects = rawlsBillingProjects map {
+          project =>
+            // TODO: throw exception if ID doesn't exist (already filtering for this, so is it necessary?)
+            val profileModel = profileApi.getProfile(UUID.fromString(project.billingProfileId.get))
             RawlsBillingProject(
```

Comment:
Is there an easier way to clone a `RawlsBillingProject` in order to add the `azureManagedAppCoordinates` information?

Reply:
Case classes have a `copy` method that may be suitable.

Reply:
Yep, copy was what I needed.